### PR TITLE
Merge tag 'v205'

### LINKED ISSUE #50: Merge with upstream to upgrade from v191 to v205

### PR DESCRIPTION
**Pull request by EtienneM**
I added two commits: 3c5b484 and 1cf03ea 

Fix #50

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -47,7 +47,7 @@ jobs:
     parameters:
       stack:
         type: enum
-        enum: ["heroku-16", "heroku-18", "heroku-20"]
+        enum: ["heroku-18", "heroku-20"]
```
Comment (curzolapierre) [inline .circleci/config.yml:50]: Nitpick: Shouldn't we change heroku stack here too?

File: .circleci/config.yml [hunk 2]
```diff
@@ -87,4 +87,4 @@ workflows:
       - hatchet:
           matrix:
             parameters:
-              stack: ["heroku-16", "heroku-18", "heroku-20"]
+              stack: ["heroku-18", "heroku-20"]
```
Comment (curzolapierre) [inline .circleci/config.yml:90]: Same as above

**Review by curzolapierre — APPROVED** [review]
LGTM

File: .circleci/config.yml (hunk 1 above)
Comment (EtienneM) [reply .circleci/config.yml:50]: This is CircleCI configuration and we don't use CIrcleCI. This is the upstream CI. We don't really care. We trust upstream that the CI is good. No CI on our side.

**Pull request merged**